openedx/features/course_experience/url_helpers.py

```diff
@@ -57,18 +135,27 @@ def get_legacy_courseware_url(course_key, usage_key, request=None):
     return redirect_url
 
 
-def get_learning_mfe_courseware_url(course_key, sequence_key=None, unit_key=None):
+def get_learning_mfe_courseware_url(
+        course_key: CourseKey,
+        sequence_key: Optional[UsageKey] = None,
+        unit_key: Optional[UsageKey] = None,
+) -> str:
     """
-    Return a str with the URL for the specified coursweare content in the Learning MFE.
+    Return a str with the URL for the specified courseware content in the Learning MFE.
 
     The micro-frontend determines the user's position in the vertical via
-    a separate API call, so all we need here is the course_key, section, and
+    a separate API call, so all we need here is the course_key, sequence, and
     vertical IDs to format it's URL. For simplicity and performance reasons,
     this method does not inspect the modulestore to try to figure out what
     Unit/Vertical a sequence is in. If you try to pass in a unit_key without
     a sequence_key, the value will just be ignored and you'll get a URL pointing
     to just the course_key.
 
+    Note that `sequence_key` may either point to a Section (ie chapter) or
+    Subsection (ie sequential), as those are both abstractly understood as
+    "sequences". If you pass in a Section-level `sequence_key`, then the MFE
+    will replace it with key of the first Subsection in that Section.
+
```

Comment:
This is a change we need to make to the Learning MFE before using `get_courseware_url` in production.